### ISSUE TITLE
Own TV scraper doesn't load IMDB info in some cases

### ISSUE THREAD

**Issue by Diskutant** [Komet/MediaElch#1616]
**Scrapers that don't work:**

 - [x] IMDb
 - [x] TMDb

**Details which are wrong / not loaded:**
Title, Original title, Rating, Votes

**MediaElch Version:**
 - [x] 2.10.3-dev (nightly)

**Operating System:**
 - [x] macOS

**Additional context:**
This seems to happen only with one particular show "Das Geheimnis des Sagala" when using the "own scraper".

Old info I already have in the nfo:
<img width="1923" alt="Bildschirmfoto 2023-07-23 um 14 46 16" src="https://github.com/Komet/MediaElch/assets/898220/99cc01c8-e6ce-4a46-9a77-9684fc7d3060">

After scraping to get updated information:
<img width="1923" alt="Bildschirmfoto 2023-07-23 um 14 47 11" src="https://github.com/Komet/MediaElch/assets/898220/4bf1d6ba-07c5-4694-a304-b3d94463c4fc">

Scraper settings (the missing info is set to IMDB):
<img width="1234" alt="Bildschirmfoto 2023-07-23 um 14 41 41" src="https://github.com/Komet/MediaElch/assets/898220/9bcad59c-6534-4d51-943c-734d0acc4366">

The scraper window:
<img width="840" alt="Bildschirmfoto 2023-07-23 um 14 41 16" src="https://github.com/Komet/MediaElch/assets/898220/e31b6411-40e2-4373-a544-e48dc93244ad">

If I use the IMDB Scraper it works better:
<img width="1879" alt="Bildschirmfoto 2023-07-23 um 14 54 18" src="https://github.com/Komet/MediaElch/assets/898220/06ab2dbc-c72d-4df6-a577-84e4c65643f1">

**Comment by bugwelle** [comment]
Thanks for reporting! I'll try to have a look at this next week. 

Fields are cleared even if the assigned scraper does not report any data. I want to move to a "merge" features sometime in the future.

I you scrape a TV show only via IMDb (not the custom scraper), does it work?

IMDb is hard to scrape. It does not support localization, so sometimes it fails to load TV shows in other languages (e.g. German in this case).

Regards,
Andre

**Comment by Diskutant** [comment]
> If you scrape a TV show only via IMDb (not the custom scraper), does it work?

yes, that works as shown in the last screenshot in the bug report. ;)
